stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -57,10 +57,13 @@ Keyword argument `debuginfo` may be one of `:source` or `:none` (default), to sp
 
 See [`@code_warntype`](@ref man-code-warntype) for more information.
 """
-function code_warntype(io::IO, @nospecialize(f), @nospecialize(t); debuginfo::Symbol=:default, optimize::Bool=false)
+function code_warntype(io::IO, @nospecialize(f), @nospecialize(t);
+                       debuginfo::Symbol=:default, optimize::Bool=false,
```

Comment:
Perhaps we should have written this with kwargs forwarding?
```suggestion
                       debuginfo::Symbol=:default, kwargs...)
```

Reply:
Yeah, that's better! I'll change it.